docs/howto/connection/cassandra.rst

```diff
@@ -0,0 +1,109 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+
+Apache Cassandra Connection
+=======================
+
+The Apache Cassandra connection type enables connection to Apache Cassandra.
+
+Default Connection IDs
+----------------------
+
+Cassandra hook and Cassandra operators use ``cassandra_default`` by default.
+
+Configuring the Connection
+--------------------------
+Host (required)
+    The host to connect to. Hosts can be provided in comma-separated string values.
```

Comment:
```suggestion
    The host to connect to. It is possible to specify multiple hosts as a comma-separated list.
```
This is not very clear since the first sentence says only one host.